docs/descheduler.md

```diff
@@ -0,0 +1,143 @@
+# Descheduler
+
+Users could divide their replicas of a workload into different clusters in terms of available resources of member clusters.
+However, the scheduler's decisions are influenced by its view of Karmada at that point of time when a new `ResourceBinding` 
+appears for scheduling. As Karmada multi-clusters are very dynamic and their state changes over time, there may be desire 
+to move already running replicas to some other clusters due to lack of resources for the cluster. This may happen when 
+some nodes of a cluster failed and the cluster does not have enough resource to accommodate their pods or the estimators 
+have some estimation deviation, which is inevitable.
+
+The karmada-descheduler will detect all deployments once in a while, default 2 minutes. In every period, it will find out 
```

Comment:
```suggestion
The karmada-descheduler will detect all deployments once in a while, every 2 minutes by default. In every period, it will find out 
```